src/app/find-help/accommodation-search/accommodation-search.component.ts

```diff
@@ -17,6 +17,8 @@ export class AccommodationSearchComponent implements OnInit {
   corePath = CorePath;
   searchCriteria: AccommodationQuery = {};
   pagination: Pageable | undefined = {};
+  @ViewChild('accomodationResultsStart', { read: ElementRef }) resultsStart!: ElementRef;
```

Comment:
Unfortunately, `ElementRef` defaults its `nativeElement` property to `any` so it's good to give it a concrete type parameter. In this case, the element is `mat-card` so nothing specific like `input` (that would be represented by `HtmlInputElement`) so:
```suggestion
  @ViewChild('accomodationResultsStart', { read: ElementRef }) resultsStart!: ElementRef<HTMLElement>;
```

Reply:
Done